hilti/runtime/include/debug-logger.h

```diff
@@ -24,14 +24,15 @@ class DebugLogger {
     bool isEnabled(std::string_view stream) { return _streams.find(stream) != _streams.end(); }
 
     void indent(std::string_view stream) {
-        if ( isEnabled(stream) )
-            _streams[stream] += 1;
+        if ( auto s = _streams.find(stream); s != _streams.end() ) {
```

Comment:
How about adding a quick check upfront if `_streams` is empty? Or is `find()` good enough in that case? (Same in `dedent`.)

Reply:
I looked into the impl of `find` with the libc++ I have on my system. There `find` uses `lower_bound` which already short-circuits for an empty list, so I'd prefer to keep this as is.